### PR TITLE
Problem: Missing contribution guidelines

### LINKED ISSUE #412: Problem: Missing contribution guidelines

### PR DESCRIPTION
**Pull request by calvinaco**
Solution: (Fix #412) Add guidelines and GitHub templates

### CODE REVIEW

**Review by eddycpc — APPROVED** [review]
LGTM

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- [ ] Have you checked your code compiles? (`yarn build`)
+- [ ] Have you included tests for any non-trivial functionality?
+- [ ] Have you checked your code passes the unit tests? (`yarn test`)
+- [ ] Have you checked your code formatting is correct? (`yarn lint:js`)
+- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`yarn audit`)
+- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
+- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-com/chain-main/blob/master/CHANGELOG.md)?
+- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-com/chain-desktip-wallet/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-com/chain-desktip-wallet/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).
```
Comment (tomtau) [inline .github/PULL_REQUEST_TEMPLATE.md:16]:
```suggestion
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-com/chain-desktop-wallet/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-com/chain-desktop-wallet/blob/master/CONTRIBUTING.md#developer-certificate-of-origin).
```

**Comment by crypto-matto** [comment]
Thanks @calvinaco for all the contents~ 
Is `SECURITY.md` missing for the moment?

**Comment by calvinaco** [comment]
> Thanks @calvinaco for all the contents~
> Is `SECURITY.md` missing for the moment?

Just added

**Review by eddycpc — APPROVED** [review]

**Pull request merged**